.github/workflows/release.yml

```diff
@@ -93,11 +93,8 @@ jobs:
           GIT_TAG=$(git describe --tags `git rev-list --tags --max-count=1` | sed "s/v//")
           docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
 
-          docker build -t dotenvlinter/dotenv-linter:${GIT_TAG} .
-          docker push dotenvlinter/dotenv-linter:${GIT_TAG}
-
-          docker build -t dotenvlinter/dotenv-linter:latest .
-          docker push dotenvlinter/dotenv-linter:latest
+          docker buildx --push --platform linux/amd64,linux/arm64 -t dotenvlinter/dotenv-linter:${GIT_TAG} .
```

Comment:
I've never tried it and I think we should only use it if it gives us some advantages instead of these three lines we have right now.
Is there anything specific you see as an advantage in using this action?

Reply:
I just meant that docker buildx is a plugin. And I'm not sure if it are already included in github-actions or not.

Reply:
That's right and that is also the only thing I'm not sure about, but as this comes included in docker when you install it "normally" via package managers I would assume it is included in `ubuntu-latest`.

Would be cool to trigger a test release to see if it is working. 
This would be nice if we use the action too to see it working.

If it is working I would be in favour of the own solution instead of pulling in a custom action, but that's also some kind of personal preference and I like to only use third party tools/libraries if it really makes sense.

Reply:
Yeah, it make sense. 
I can suggest one working-around for test purpose: take a similar step in the `build` pipeline without `pushing`. We will see that everything works correctly, and you can remove the "temporary" step from CI then.

Reply:
> Yeah, it make sense. I can suggest one work-around for test purpose: take a similar step in the `build` pipeline without `pushing`. We will see that everything works correctly, and you can remove the "temporary" step from CI then. (It doesn't require merge before testing, you will see results in checks)

It would be great to update the `docker_build` step in `ci.yml`.

Reply:
oh i forgot we already have `ci` step with `docker_build`. In this case I agree, we can just update the existing step

Reply:
@mstruebing Please do it 🙏